sync.py

```diff
@@ -107,65 +113,67 @@ def yes_or_no(question):
 	# if the find process itself would return an error code > 0 though.
 	# So a better solution is probably suitable later.
 	if e.returncode != 1:
-		print "An error occured with the find subprocess!"
-		print "returncode", e.returncode
-		print "output", e.output
+		print('An error occured with the find subprocess!')
+		print('returncode', e.returncode)
+		print('output', e.output)
 
 if isinstance(perm_output, str):
-	print "Some files have wrong permissions:"
-	print perm_output
+	print('Some files have wrong permissions:')
+	print(perm_output)
 
-	choice = yes_or_no("Do you want to continue syncing anyway? ")
+	choice = yes_or_no('Do you want to continue syncing anyway? ')
 
 	if choice:
-		print "All right, will sync anyway."
+		print('All right, will sync anyway')
 		wrong_perm = True
 	else:
-		print "All right, aborting."
+		print('All right, aborting.')
 		sys.exit()
 
 else:
-	print "Everything looks OK. Continuing with rsync."
+	print('Everything looks OK. Continuing with rsync.')
 
 
-# Step 5. Sync our destignated folders.
+# Step 5. Sync our designated folders.
+print('Syncing directories:\n{}\n{}'.format(src_root_path, src_containers_path))
 
-excludes = "--exclude=*.swp --exclude=irma3/"
-rsync_cmd = "/bin/rsync -avzP --omit-dir-times {0} --log-file={1} {2} {3} {4}@{5}:{6}".format(excludes,
+excludes = '--exclude=*.swp --exclude=irma3/'
+rsync_cmd = '/bin/rsync -avzP --relative --omit-dir-times {0} --log-file={1} {2} {3} {4}@{5}:{6}'.format(excludes,
                                                                                                rsync_log_path,
                                                                                                src_root_path,
                                                                                                src_containers_path,
                                                                                                user,
                                                                                                host,
                                                                                                dest)
 # TODO: Do this cleaner?
-dry_cmd = "/bin/rsync --dry-run -avzP --omit-dir-times {0} {1} {2} {3}@{4}:{5}".format(excludes, src_root_path, src_containers_path, user, host, dest)
+if args.dryrun:
+	dry_cmd = '/bin/rsync --dry-run -avzP --relative --omit-dir-times {0} {1} {2} {3}@{4}:{5}'.format(excludes, src_root_path, src_containers_path, user, host, dest)
 
-# First doing a dry-run to confirm sync.
```

Comment:
Maybe we can remove this comment since it isn't always the first thing we do.

Reply:
Done!